.gitignore

```diff
@@ -0,0 +1,7 @@
+# macOS environment
+.DS_Store
+# Xcode artifacts
```

Comment:
nit:

Could you please put the blank lines to separate logical blocks of code?

```suggestion
.DS_Store

# Xcode artifacts
```

to improve readability.

Reply:
added